spark-k8s/versions.py

```diff
@@ -1,31 +1,13 @@
 versions = [
-    {
-        "product": "3.5.2",
-        "java-base": "17",
-        "java-devel": "17",
-        "python": "3.11",
-        "hadoop": "3.3.6",  # Hadoop version defined in ../hbase/versions.py to reduce build time and disk requirements
-        "hbase": "2.6.1",  # current Stackable LTS version
-        "aws_java_sdk_bundle": "1.12.367",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/3.3.6
-        "azure_storage": "7.0.1",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-azure/3.3.6
-        "azure_keyvault_core": "1.0.0",  # https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage/7.0.1
-        "jackson_dataformat_xml": "2.15.2",  # https://mvnrepository.com/artifact/org.apache.spark/spark-core_2.13/3.5.1
-        "stax2_api": "4.2.1",  # https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.15.2
-        "woodstox_core": "6.5.1",  # https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml/2.15.2
-        "vector": "0.46.1",
-        "jmx_exporter": "1.2.0",
-        "tini": "0.19.0",
-        "hbase_connector": "1.0.1",
-    },
     {
         "product": "3.5.5",
         "java-base": "17",
         "java-devel": "17",
         "python": "3.11",
-        "hadoop": "3.3.6",  # Hadoop version defined in ../hbase/versions.py to reduce build time and disk requirements
-        "hbase": "2.6.1",  # current Stackable LTS version
-        "aws_java_sdk_bundle": "1.12.367",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/3.3.6
-        "azure_storage": "7.0.1",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-azure/3.3.6
+        "hadoop": "3.4.1",  # Current Stackable LTS version. Source of the AWS and Azure artifacts to Spark's classpath.
+        "hbase": "2.6.2",  # Current Stackable LTS version. Used to build the HBase connector.
+        "aws_java_sdk_bundle": "1.12.720",  # https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/3.4.1
```

Comment:
Just curious, [this](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/aws_sdk_upgrade.html) mentions:

> `aws-java-sdk-bundle-1.12.x.jar` becomes `bundle-2.x.y.jar`

And I do see: `bundle-2.24.6.jar` in `/stackable/hadoop-3.4.1/share/hadoop/tools/lib/`.

I only see `aws-java-sdk-bundle-1.12.367.jar` for hadoop 3.3.6.

Reply:
wow. good catch